eversql/README.md

```diff
@@ -0,0 +1,42 @@
+# EverSQL
+
+## Overview
+
+[EverSQL][1] is the fastest way to automatically speed up your database and optimize SQL queries, providing automatic SQL tuning and indexing for developers, DBAs, and DevOps engineers.
```

Comment:
```suggestion
[EverSQL][1] is a way to speed up your database and optimize SQL queries, providing automatic SQL tuning and indexing for developers, DBAs, and DevOps engineers.
```

Reply:
Some edits here to remove marketing-focused language and redundancy. Please see the contributing guidelines for Datadog documentation:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#general-principles